src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -1069,41 +1183,36 @@ public void rename(String applicationName, String newName) {
     }
 
     @Override
-    public StartingInfo restartApplication(String applicationName) {
+    public void restartApplication(String applicationName) {
         stopApplication(applicationName);
-        return startApplication(applicationName);
+        startApplication(applicationName);
```

Comment:
Take a look at this https://v3-apidocs.cloudfoundry.org/version/3.92.0/#restart-an-app
Can we reduce the number of network calls if we use this?
It will require 1 call for getting the id + another for the restart.
Actually, after taking a look at the `cf-java-client`, they don't have the `restart` action. A contribution is needed if we want to use this. But it is still an experimental resource, so maybe we shouldn't until it's made GA

Reply:
It could be contributed when it is GA